### PR TITLE
Changes for Animation when Happiness stats alters

### PR DESCRIPTION
**Pull request by benjaminCMD**
I have been been to somewhat fix happiness. The async functions presented inside Pet.jsx are still buggy so I commented them out for the time. Growth Level still defaults to 1, so right now that represents the Adult Stage for the time being.
image

This pull request is simply a continuation of #221, so that one can be closed, since that branch is no longer gonna get merged

### CODE REVIEW

**Review by PinkSylvie — APPROVED** [review]
Thank you for the fixes, it works perfectly now. Great work!

**Comment by PinkSylvie** [comment]
Please fix the conflicting files so you can merge

**Review by irsaris — APPROVED** [review]

File: components/Pet.jsx [hunk 1]
```diff
@@ -77,9 +76,8 @@ class Pet extends Component {
       pettype: 'frog',
       images: [],
       startDate: new Date("2024-03-16"), //Date the pet was created,  we need to get the info from pet creation
-      examDate: new Date("2024-05-18"), // Date the exam is due , we need to get the info from pet creation, please implement this
```
Comment (FrancesC8) [inline components/Pet.jsx:80]: I noticed that a lot of changes from main are being deleted. Make sure to be aware of what changes you want to merge.

**Comment by PinkSylvie** [comment]
This still has conflicts but I see that you made another pull request that is related to this and the stages of growth so I'm gonna close this pull request.

**Pull request closed without merging**